libmamba/src/core/environments_manager.cpp

```diff
@@ -207,9 +207,9 @@ namespace mamba
         return p;
     }
 
-    fs::u8path EnvironmentsManager::get_environments_txt_file(const fs::u8path& home) const
+    fs::u8path EnvironmentsManager::get_environments_txt_file(const fs::u8path& user_data_dir_in) const
```

Comment:
Agree. It would be better if we could still see the environments (previously) created by `conda`/`mamba`.

But yes, in a perfect world, this would belong there.

If we decide to leave this be `home / ".conda" / "environments.txt"`, we may want to add a comment explaining this discrepancy between the other config files.

Another option could be to read from both the conda and the mamba `environments.txt` but only write to the one in `user_data_dir`.

Reply:
We should continue writing to that file so that you can use Conda with envs created with Mamba .